.gitignore

```diff
@@ -1,10 +1,9 @@
-# GoDot auto generated files
+# Godot auto generated files
 platform/server/logo.h
 platform/android/logo.h
 platform/bb10/logo.h
 platform/iphone/logo.h
 platform/javascript/logo.h
-platform/nacl/logo.h
 platform/osx/logo.h
 platform/windows/logo.h
 platform/x11/logo.h
```

Comment:
It wouldn't hurt yeah. Most important was to fix `s/GoDot/Godot/`. I'm still shivering from it :P